io-engine/src/rebuild/rebuild_job_backend.rs

```diff
@@ -402,17 +445,7 @@ impl RebuildJobBackend {
             block_size: self.descriptor.block_size,
             tasks_total: self.task_pool.total as u64,
             tasks_active: self.task_pool.active as u64,
-        };
-
-        info!(
-            "State: {state}, Nexus: {nex}, Src: {src}, Dst: {dst}: {res:?}",
-            state = self.state(),
-            nex = self.nexus_name,
-            src = self.src_uri,
-            dst = self.dst_uri,
-        );
-
-        res
+        }
```

Comment:
The stats are still printed in logs, I just wanted to avoid duplicate messages in  certain situations. I moved logging of the stats in other places from where get_stats() is called.